perf/istio/gateway-bouncer/setup.sh

```diff
@@ -0,0 +1,46 @@
+#/bin/bash
+set -ex
+
+NAMESPACE=${NAMESPACE:?"New/existing NAMESPACE to install this scenario into"}
+
+function install_gateway_bouncer() {
+  local DIRNAME="${1:?"output dir"}"
+  local OUTFILE="${DIRNAME}/gateway_bouncer.yaml"
+  local INGRESS_IP=""
+
+  helm -n $NAMESPACE template \
+    --set namespace=$NAMESPACE \
+    . > "${OUTFILE}"
+
+  if [[ -z "${DRY_RUN}" ]]; then
+    kubectl -n $NAMESPACE apply -f "${OUTFILE}"
+
+    # Waiting until LoadBalancer is created and retrieving the assigned
+    # external IP address.
+    while : ; do
```

Comment:
One concern here is if a LB isn't being created, will this just spin forever with no indication?

Or does the `set -x` show what is being done?

Reply:
Yes, -x provides quite a bit of feedback, try running it -- pretty self explanatory. Hopefully in the future we can streamline the installation process, so we won't have to rely on a bunch of ad-hoc scripts.